app/frontend/src/assets/scss/style.scss

```diff
@@ -480,6 +480,25 @@ a,
   }
 }
 
+// allow dropdowns to continue expanding downwards within a modal
+.formio-dialog-content {
+  .component-edit-container {
+    overflow: visible;
+  }
+}
+
+.choices.form-group.formio-choices {
+  .choices__list.choices__list--dropdown {
+    overflow: auto;
+  }
+}
+
+.choices__list.choices__list--dropdown {
+  div {
```

Comment:
The rules are good. cant really do it the way i hoped or use javascript or override much of formio. so this is great.

you can make this final div selector more specific. `div.choices__list`
and use the sass nesting to prevent all of these 'choices' dropdowns being affected (including those outside of this form-builder modal). so do:
```
.formio-dialog-content {
  .component-edit-container {
    overflow: visible;

    .choices__list.choices__list--dropdown {
      overflow: auto;

      div.choices__list {
        max-height: 175px;
      }
    }
  }
}
```

Reply:
Made changes to the ordering and structure of the SCSS in the new commit